File: app/src/main/java/uk/co/mezpahlan/jamesedwardtaylorstheguardian/theguardian/TheGuardianApplication.java
package uk.co.mezpahlan.jamesedwardtaylorstheguardian.theguardian;

import android.app.Application;

import uk.co.chrisjenx.calligraphy.CalligraphyConfig;
import uk.co.mezpahlan.jamesedwardtaylorstheguardian.R;

/**
 * Application for TheGuardian. Sets up a custom typeface throughout the app.
 */
public class TheGuardianApplication extends Application {

    @Override
    public void onCreate() {
        super.onCreate();
        CalligraphyConfig.initDefault(new CalligraphyConfig.Builder()
                .setDefaultFontPath("fonts/Merriweather-Regular.ttf")
                .setFontAttrId(R.attr.fontPath)
                .build()
        );
    }
}

File: app/src/main/kotlin/uk/co/mezpahlan/oldtimerag/theguardian/TheGuardianApplication.kt
package uk.co.mezpahlan.oldtimerag.theguardian


import android.app.Application
import org.koin.android.ext.android.startKoin
import uk.co.mezpahlan.oldtimerag.theguardian.di.appModules

/**
 * Application for TheGuardian. Sets up a custom typeface throughout the app.
 */
class TheGuardianApplication : Application() {

    override fun onCreate() {
        super.onCreate()

        startKoin(this, appModules)
    }
}
